packages/db/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.9"
```

Comment:
made this in the interim, not sure if you have another setup on the other repos?

Reply:
That sounds good to me.